src/backends/interflop-mca/interflop_mca.c

```diff
@@ -209,34 +226,44 @@ static __float128 _noise_binary128(const int exp) {
 
 /* Macro function that adds mca noise to X
    according to the virtual_precision VIRTUAL_PRECISION */
-#define _INEXACT(X, VIRTUAL_PRECISION)                                         \
+#define _INEXACT(X, VIRTUAL_PRECISION, CTX)                                    \
   {                                                                            \
     if (_MUST_NOT_BE_NOISED(*X, VIRTUAL_PRECISION)) {                          \
       return;                                                                  \
     } else {                                                                   \
       const int32_t e_a = GET_EXP_FLT(*X);                                     \
```

Comment:
Since e_a is not used in the absolute mode, it could be hoisted inside the following `if`.

Reply:
Ok.
Solved.